scripts/build/VSImport-Samples.ps1

```diff
@@ -14,15 +15,18 @@ $ErrorActionPreference = "Stop"
 
 & $PSScriptRoot\unzip.ps1 -zipFile $zipFile -outputDir $tempDir
 
-$XCodeProjects = gci -recurse $tempDir\*.xcodeproj
+$XCodeProjects = gci -recurse $PSScriptRoot\..\..\samples\*.xcodeproj
+$XCodeProjects += gci -recurse $PSScriptRoot\..\..\tests\testapps\*.xcodeproj
```

Comment:
..why build? There's no xcodeprojs under build.

---
In reply to: [103062579](https://github.com/Microsoft/WinObjC/pull/2097#discussion_r103062579) [](ancestors = 103062579,103061909,103060288)